src/test/javascript/portal/details/StylePanelSpec.js

```diff
@@ -101,6 +101,38 @@ describe("Portal.details.StylePanel", function() {
             expect(getParameterByNameFromUrlString(urlString, "VERSION")).toEqual("1.1.0");
         });
 
+        it("should include numColorBands if it is defined", function() {
+
+            var layer = {
+                extraLayerInfo: {numColorBands: '24'},
```

Comment:
Why is it hard coded to '24' here and line 112? numColorBands can be anything between 2 and 253 I think.

Reply:
Just for the purposes of Jasmine tests. The actual value is determined from the config